hyp3_sdk/hyp3.py

```diff
@@ -120,71 +120,131 @@ def _refresh_batch(self, batch: Batch):
     def _refresh_job(self, job: Job):
         return self._get_job_by_id(job.job_id)
 
-    def submit_job_dict(self, job_dict: dict, name: Optional[str] = None, validate_only: bool = False) -> Job:
-        if name is not None:
-            if len(name) > 20:
-                raise ValidationError('Job name too long; must be less than 20 characters')
-            job_dict['name'] = name
+    def submit_prepared_jobs(self, prepared_jobs: Union[dict, List[dict]]) -> Batch:
+        """Submit a prepared job dictionary, or list of prepared job dictionaries
+
+        Args:
+            prepared_jobs: A prepared job dictionary, or list of prepared job dictionaries
+
+        Returns:
+            A Batch object containing the submitted job(s)
+        """
+        if isinstance(prepared_jobs, dict):
+            payload = {'jobs': [prepared_jobs]}
+        else:
+            payload = {'jobs': prepared_jobs}
 
-        payload = {'jobs': [job_dict], 'validate_only': validate_only}
         response = self.session.post(urljoin(self.url, '/jobs'), json=payload)
         try:
             response.raise_for_status()
         except HTTPError:
             raise HyP3Error('Error while submitting job to HyP3')
-        return Job.from_dict(response.json()['jobs'][0])
 
-    def submit_autorift_job(self, granule1: str, granule2: str, name: Optional[str] = None) -> Job:
+        return Batch(response.json()['jobs'])
```

Comment:
Since this function will optionally submit multiple jobs, we need to return a batch object for the general case. Or, we could conditionally return.